NewRelicBrowser.php

```diff
@@ -1,392 +1,456 @@
+
+			wp_enqueue_style( 'rtp_relic_css', plugins_url( '/css/rtp-new-relic.css', __FILE__ ), array(), RTP_RESOURCE_VERSION );
+			wp_enqueue_style( 'rtp_relic_ui_css', plugins_url( '/css/jquery-ui.css', __FILE__ ), array(), RTP_RESOURCE_VERSION );
 		}
 
 		/**
-		 * Validate user form
-		 * @param array $relic_user_data
+		 * Validate user form.
+		 *
+		 * @param array $relic_user_data Relic User Data.
+		 *
 		 * @return array
 		 */
-		function rtp_relic_validate_form( $relic_user_data )
-		{
-
-			$relic_valid = true;
+		public function rtp_relic_validate_form( $relic_user_data ) {
+			$relic_valid         = true;
 			$relic_error_message = '';
+
 			if ( isset( $relic_user_data['rtp-relic-form-name'] ) ) {
 				$form_name = sanitize_text_field( $relic_user_data['rtp-relic-form-name'] );
 			}
 
-			if ( 'rtp-get-browser' == $form_name ) {
+			if ( 'rtp-get-browser' === $form_name ) {
 				/* get browser list form */
-				if ( ( '' == $relic_user_data['rtp-user-api-key'] ) ) {
-					$relic_valid = false;
-					$relic_error_message = __( 'All fields are required.', 'rt-new-relic' );
+				if ( ( '' === $relic_user_data['rtp-user-api-key'] ) ) {
+					$relic_valid         = false;
+					$relic_error_message = esc_html__( 'All fields are required.', 'rt-new-relic' );
 				}
-			} else if ( 'rtp-select-browser' == $form_name ) {
+			} elseif ( 'rtp-select-browser' === $form_name ) {
 				/* select browser application form */
-				if ( '' == $relic_user_data['rtp-selected-browser-id'] ) {
-					$relic_valid = false;
-					$relic_error_message = __( 'Select atleast one application.', 'rt-new-relic' );
+				if ( '' === $relic_user_data['rtp-selected-browser-id'] ) {
+					$relic_valid         = false;
+					$relic_error_message = esc_html__( 'Select atleast one application.', 'rt-new-relic' );
 				}
-			} else if ( 'rtp-add-account' == $form_name ) {
+			} elseif ( 'rtp-add-account' === $form_name ) {
 				/* add new user account form */
-				if ( ( '' == $relic_user_data['relic-account-email']) || ( '' == $relic_user_data['relic-first-name']) || ( '' == $relic_user_data['relic-last-name']) || ( '' == $relic_user_data['relic-account-name']) ) {
-					$relic_valid = false;
-					$relic_error_message = __( 'All fields are required.', 'rt-new-relic' );
-				} else if ( ! filter_var( $relic_user_data['relic-account-email'], FILTER_VALIDATE_EMAIL ) ) {
-					$relic_valid = false;
-					$relic_error_message = __( 'Not a valid email address', 'rt-new-relic' );
-				} else if ( ( ! preg_match( '/^[a-zA-Z ]*$/', $relic_user_data['relic-first-name'] )) || ( ! preg_match( '/^[a-zA-Z ]*$/', $relic_user_data['relic-last-name'] )) ) {
-					$relic_valid = false;
-					$relic_error_message = __( 'Name should contain letters only', 'rt-new-relic' );
+				if ( ( '' === $relic_user_data['relic-account-email'] ) || ( '' === $relic_user_data['relic-first-name'] ) || ( '' === $relic_user_data['relic-last-name'] ) || ( '' === $relic_user_data['relic-account-name'] ) ) {
```

Comment:
Use `empty()` instead of comparing empty string.

Reply:
i solved this issue. please review changes.